scripts/crc-deploy.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+set -e
+
+# This script manages the deployment of downstream olm on a local CRC cluster
+# It will deploy the local images: olm:test and opm:test
+# Both built with make build/olm-container and build/registry-container respectively
+
+echo "Deploying OLM to CRC"
+
+# Set kubeconfig to CRC if necessary
+export KUBECONFIG=${KUBECONFIG:-${HOME}/.crc/machines/crc/kubeconfig}
+KUBE_ADMIN_PASSWORD=$(tr -d '\n' < "${HOME}/.crc/machines/crc/kubeadmin-password")
```

Comment:
IIRC there's a dedicated `crc console --credentials` command that might be better to parse?

This is what I have locally as an alias:

```console
$ alias crc_l
crc_l='eval $(crc console --credentials | cut -d" -f2- | grep -v developer | tr -d ")'
```

Reply:
Oh, found an even nicer way: `crc console --credentials -o json | jq -r .clusterConfig.adminCredentials.password`